src/app/shared/models/models.component.ts

```diff
@@ -509,11 +587,11 @@ export class ModelsComponent implements OnInit, OnDestroy {
     }
   };
 
-  classifierTreeOnSelect = node => {
+  classifierTreeOnSelect = (node) => {
     this.stateHandler.Go('classification', { id: node.id });
   };
 
-  classificationFilterChange = val => {
+  classificationFilterChange = (val) => {
```

Comment:
If in this area, might as well change this to `classificationFilterChange(val) { }` instead